### PR TITLE
Copy in the js, not mjs for js-cookie to fix swagger-ui

### PR DESCRIPTION
**Pull request by ahobson**
Tiny little change to try and fix swagger-ui

### CODE REVIEW

File: scripts/copy-swagger-ui [hunk 1]
```diff
@@ -5,4 +5,4 @@
 # will need to be manually updated based on node_modules/swagger-ui-dist/index.html
 # if it ever changes.
 cp node_modules/swagger-ui-dist/{*.js,*.css,*.png} public/swagger-ui
-cp node_modules/js-cookie/dist/js.cookie.mjs public/swagger-ui
```
Comment (donaldthai) [inline scripts/copy-swagger-ui:8]: suggestion: Could we also update the js.cookie.js path of the other html files (internal.html, admin.html, api.html and ghc.html) in the `public/swagger-ui` directory. Noticed that the other html files are using the wrong path for that cookie library. Should be using `<script src="/swagger-ui/js.cookie.js"></script>`, not `<script src="/swagger-ui/dist/js.cookie.js"></script>`.

Comment (ahobson) [reply scripts/copy-swagger-ui:8]: Good catch

File: public/swagger-ui/admin.html [hunk 2]
```diff
@@ -69,7 +69,7 @@
 
 <script src="/swagger-ui/swagger-ui-bundle.js"> </script>
 <script src="/swagger-ui/swagger-ui-standalone-preset.js"> </script>
-<script src="/swagger-ui/dist/js.cookie.mjs"></script>
+<script src="/swagger-ui/dist/js.cookie.js"></script>
```
Comment (donaldthai) [inline public/swagger-ui/admin.html:72]: I think the path should be `/swagger-ui/js.cookie.js` which is also the same path in `prime.html`

Comment (ahobson) [reply public/swagger-ui/admin.html]: 🤦 Thank you, fixed

**Comment by shimonacarvalho** [comment]
What does this fix? It sounds like maybe this takes care of an error in the swagger ui? 
Which url, what is the expected error/change in behavior?
That will help me confirm the fix.

**Comment by donaldthai** [comment]
> 

When you're running the app, you can access the swagger UI page for each API. For example, swagger UI page for the GHC can be access by the path `http://officelocal:3000/ghc/v1/docs`.

**Review by donaldthai — APPROVED** [review]
👍

**Pull request merged**